save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphView.kt

```diff
@@ -208,8 +227,10 @@ val fossGraph: FC<FossGraphViewProps> = FC { props ->
                 div {
                     className = ClassName("col-xl col-md-6 mb-4")
                     val progress = vulnerability.progress
-                    val color = if (progress < 51) {
+                    val color = if (progress < 34) {
                         Color.GREEN.hexColor
+                    } else if (progress < 67) {
```

Comment:
Suggestion to move these constants to `private const val`

Reply:
This variable depends on the class variable, it cannot be made into a constant.

Reply:
I mean constants like `34` and `67`